metabase/metabase
# Use built-in functions for exclude date filters instead of temporal buckets

Epic https://github.com/metabase/metabase/issues/50238

Uses built-in MBQL functions for exclude date filters instead of relying on inline temporal bucketing. Existing expressions are supported on read.

How to verify:
- New -> Question -> Orders
- Filter -> Created At -> Exclude
- Try all options with different values. They should work as before.
- Try going to the expression editor after you create an expression using the filter widget. You should now see MBQL functions being called; previously the expression editor broke for such expressions.
